### PR TITLE
Create pharmacology.jsonld

### PR DESCRIPTION
**Pull request by lzehl**
related to #276

### CODE REVIEW

File: instances/technique/pharmacology.jsonld [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "'Pharmacology' is an experimental approach in which the composition, properties, functions, sources, synthesis and design of drugs (any artificial, natural, or endogenous molecule) and their biochemical or physiological effect (normal or abnormal) on a cell, tissue, organ, or organism are studied. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Pharmacology)]",
+  "description": null,
+  "interlexIdentifier": "http://uri.interlex.org/base/ilx_0108784",
+  "knowledgeSpaceLink": null,
+  "name": "pharmacology",
+  "preferredOntologyIdentifier": "http://edamontology.org/topic_0202",
+  "synonym": [
+    "computational pharmacology",
+    "pharmacoinformatics"
```
Comment (UlrikeS91) [inline instances/technique/pharmacology.jsonld]: Those are narrow synonyms - remove.

**Review by UlrikeS91 — CHANGES_REQUESTED** [review]

File: instances/technique/pharmacology.jsonld (hunk 1 above)
Comment (lzehl) [reply instances/technique/pharmacology.jsonld]: @UlrikeS91 I agree but the ontologies do not... should we anyway remove them?

Comment (lzehl) [reply instances/technique/pharmacology.jsonld]: I just removed them. I had the same issue with them (and in interlex those are also classified as narrow, not exact)

Comment (lzehl) [reply instances/technique/pharmacology.jsonld]: And actually the EDAM lists them twice (once under narrow and once under synonyms) ... @UlrikeS91 good to go from my side with synonyms set to null for us

**Review by UlrikeS91 — APPROVED** [review]

**Pull request merged**